DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/fTSQuerySearchOperators/metadata.json

```diff
@@ -0,0 +1,131 @@
+{
+    "Slug": "fts-query-search-operators",
+    "SourceFileName": "FTSQuerySearchOperators.java",
+    "Title": "FTS Query - Operators",
+    "DescriptionLines": [
+        "**Full Text Search query** allows searching for documents containing particular term(s) within a block of text.",
+        "",
+        "**You can customize the logical operators** between the searched terms.",
+        "",
+        "Upon query, an auto-index is created on the searched document fields."
+    ],
+
+    "Assets": [
+        {
+            "Title": "Operation in queries - Book",
+            "Type": "link",
+            "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/9-querying-in-ravendb#operations-in-queries"
```

Comment:
As discussed when you first started to work on the demos 
Pls align the `':'`

Reply:
Done

Reply:
"Pls keep the same order as in the C# demos.json file
The file is located in Controlers\Demos" - also Done